### PR TITLE
[Filebeat] kafka.log fileset change type of field class to keyword

### PR DESCRIPTION
**Pull request by ruflin**
Remove field kafka.log.trace.full from events.

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -72,6 +72,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Migrate Elasticsearch audit logs fields to ECS {pull}10352[10352]
 - Several text fields in the Logstash module are now indexed as `keyword` fields with `text` multi-fields (ECS). {pull}10417[10417]
 - Several text fields in the Elasticsearch module are now indexed as `keyword` fields with `text` multi-fields (ECS). {pull}10414[10414]
+- Remove field `kafka.log.trace.full` from kafka.log fielset. {pull}10398[10398]
+- Change field `kafka.log.class` for from kafka.log fielset from text to keyword. {pull}10398[10398]
```
Comment (ycombinator) [inline CHANGELOG.next.asciidoc]: Typos: should be `fieldset`.

Comment (ruflin) [reply CHANGELOG.next.asciidoc]: you mean fileset? :-D

Comment (ycombinator) [reply CHANGELOG.next.asciidoc]: LOL, yes.

**Comment by liabio** [comment]
why remove kafka.log.trace.full in version 7.0，I get part trace log not all trace log

**Pull request merged**